.github/actions/slack/README.md

```diff
@@ -0,0 +1,21 @@
+# Slack
+
+A GitHub Action for sending slack notifications. For more information, see slack's [documentation](https://api.slack.com/messaging/webhooks). In particular, note that the channel and workspace are set by the webhook URL, not directly by this action.
+
+## Usage
+
+```yaml
+name: My Action
+
+on:
+  push:
+
+jobs:
+  share-love:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ./.github/actions/slack
+        with:
+          WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+          MESSAGE: "I love you guys."
```

Comment:
Why do you pass the inputs in upper case, even though they are declared as lower case?

Reply:
This is not a decision made here.  The action is copied directly from nns-dapp (which in turn was copied from II).  We should really publish it so that we don't have to copy it all over the place.  But there is a reason why the actions aren't shared internally:

It doesn't seem to be possible [to reference an action defined in another dfinity repo](https://github.com/actions/checkout/issues/95).  We could make a dedicated repo, called `dfinity/github-actions` or some such, and clone it in other repos before use.  I'm not sure that works but we can try.

Reply:
Can you add a link to the original here in the file?

Reply:
Sure thing.

Reply:
Done.  I have added the source as a `SOURCE.md` so that importing changes doesn't require adding the line.

Reply:
I don't see any `SOURCE.md`.

Reply:
I was checking something locally.  Pushed.